### ISSUE TITLE
Moving front-end to React

### ISSUE THREAD

**Issue by ashley** [boosala/boosala#9]
* Reuse code for the front-end to make a react app
* Checkout branch `react` for code
* Let each stage of the front-end be a component under the directory `./src/Components/`

**Comment by billymiller-bbkillc** [comment]
Delegating this to Bubba for implementation. Created sub-issue at #13 with full details for the React refactoring work.

**Comment by billymiller-bbkillc** [comment]
⚠️ **Status from Ashley:** This is a technical task outside my domain. I've created a follow-up issue (#14) for the development team to handle. Ready for assignment to a frontend engineer.

**Comment by billymiller-bbkillc** [comment]
Analyzed the React refactor task. This involves:\n\n✅ **What I found:**\n- Branch `react` exists with starter code\n- Goal: componentize front-end under `./src/Components/`\n- ~2 other assignees (Saïd Ziani) also involved\n\n📋 **Recommendation:** This is a technical implementation task that needs hands-on development work. I'd suggest:\n\n1. Someone grabs the `react` branch and maps out the component structure\n2. Break down into smaller sub-tasks per component\n3. Use this issue for tracking/coordination\n\nI'm happy to help with planning and code review, but someone needs to handle the implementation work. Are we waiting on a specific person to start, or should we pair on this?\n\nLet me know how to best support! 💪
